### PR TITLE
Update entry point exports

### PR DESCRIPTION
**Pull request by bvaughn**
*Note that the first commit in this branch is to alpha-sort the exports for entry points to make it easier to compare between them when reviewing which file exports which APIs.

The following APIs have been added to the `react` stable entry point:
* `SuspenseList`
* `startTransition`
* `unstable_createMutableSource`
* `unstable_useMutableSource`
* `useDeferredValue`
* `useTransition`

The following APIs have been added or removed from the `react-dom` stable entry point:
* `createRoot`
* `unstable_createPortal` (removed)

The following APIs have been added to the `react-is` stable entry point:
* `SuspenseList`
* `isSuspenseList`

The following feature flags have been changed from experimental to true:
* `enableLazyElements`
* `enableSelectiveHydration`
* `enableSuspenseServerRenderer`

# Todo
- [x] Update tests once artifacts are built

### CODE REVIEW

File: packages/react/index.classic.fb.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export {
   StrictMode,
   Suspense,
   SuspenseList,
-  SuspenseList as unstable_SuspenseList,
+  SuspenseList as unstable_SuspenseList, // TODO: Remove once call sights updated to SuspenseList
```
Comment (acdlite) [inline packages/react/index.classic.fb.js:20]: nit: callsites

File: packages/react-devtools-scheduling-profiler/src/hooks.js [hunk 2]
```diff
@@ -7,11 +7,7 @@
  * @flow
  */
 
-import {
-  unstable_createMutableSource as createMutableSource,
-  unstable_useMutableSource as useMutableSource,
-  useLayoutEffect,
-} from 'react';
+import {createMutableSource, useMutableSource, useLayoutEffect} from 'react';
```
Comment (rickhanlonii) [inline packages/react-devtools-scheduling-profiler/src/hooks.js:10]: Are we keeping these prefixed with `unstable_`?

**Review by rickhanlonii — APPROVED** [review]

File: packages/react-devtools-scheduling-profiler/src/hooks.js (hunk 2 above)
Comment (bvaughn) [reply packages/react-devtools-scheduling-profiler/src/hooks.js:10]: Yes. That was the agreement, _but_ DevTools are built with the _experimental_ release channel.

**Comment by Jack-Works** [comment]
Does that mean CM is going to released formally? 👀

**Comment by NE-SmallTown** [comment]
see https://github.com/facebook/react/issues/13206#issuecomment-841577602 and https://github.com/facebook/react/issues/21525#issuecomment-844126488

**Pull request merged**